docs/release-notes/release-notes-1.0.0.md

```diff
@@ -0,0 +1,104 @@
+
+In order to verify the release, you'll need to have `gpg` or `gpg2` installed on your system. Once you've obtained a copy (and hopefully verified that as well), you'll first need to import the keys that have signed this release if you haven't done so already:
+
+```
+-----BEGIN PGP PUBLIC KEY BLOCK-----
+
+mQGNBGTLayEBDADUoSIR+C+GNqp1RygG6fSijuvqlIQ8np/ij1x1Fzvprk4FxBIp
+tLfDxdeCmjJ8OxojfVLRuW6hgxd2kQmGJBv2MfIMRXWpvGbQBttuprbrT2/BE2F/
+f4CkyZCldFUS915isR7mkZ09u72rr4feHyr5Ff8e9GwKoYGq3SCD9KrdiPw2H8AB
+TE8usZ6Eqh+O0cy9Lf3mOi54hwN1goVC2v6ixcle24MU30gC/NgYk2LfjmbQhBJ2
+v+cRUtvDLxAIEOf2FLYv78WqVYIyCny3cohlMsi75egKoy+Hn/B3NCcHJi74b/Fu
+QXffVPuM/g+cbV8j6q4ZbPwR19BdLNR9fKxPybsYndM+yFccIFUNIAvLjo0cndlu
+4qTtU6opaYVgHSl/iek5jwTJlVT7c5Lrm9X2FJhdB6saefc5Rk1ctvfAsmDM+M49
+Ho2Jd1EXnwk+6UdZEE2sVqox5lUaIKSCeOMMwUJt5IbIW6ocwMAu08OCei49MXEj
+C9CptJAz6Xf+Xt8AEQEAAbQlQ3J5cHRvcG93ZXIgPHJlbGVhc2VAY3J5cHRvcG93
+ZXIuZGV2PokB1AQTAQoAPhYhBFwmv+xsJGalKNVVHNBax09ol25SBQJky2shAhsD
+BQkDwmcABQsJCAcCBhUKCQgLAgQWAgMBAh4BAheAAAoJENBax09ol25SOLYMAMsG
++yiCcikPuvdojzuCpGVXjFNYP2PpO9LswvvkALVfIf4fNgJWcNQKrvoYMmX5Ia6v
+HXX3SHcDfWW/sw7TTRLkk+Tz00ZjsVdToAFGmVDHd8nVngyusgBuBw/oQWUKYnrU
++iwY7llbleLlUK+2yJgLtkjkCH4p91tocJfc3SsWC+IXxOy1n0lyCPeJsm6l1+kl
+S4M0oborkzxJR6Fufj4q/cgBJhbpvU8fuu10Kugf+MXYO7XP2UyCAmJzfgoD+Huk
+JJDlQJhBtioUaJ2jBBXEITzhNQ9DA/x+iUN7rQFNhKsMeczEErDmKucGrPSqQKiG
+Hz7c20lVnrT5EGdEL1bBPzSY7SJKkzn0rgsAxPHFxZlSkY/AFhn+/py/0+Tk1Rjq
+7ullWCiAtBpOtIySGv8wnJHNSXVRE3kU5vPOXeF0YVqhjgIyVK+1jiXErSRQaSn1
+pkpeNavFQQ0CrJlEeKFWicvBkbSfuhDArDeyq8PMdGCaD2pvMEla5VvTi1A3PLkB
+jQRky2shAQwAt01gHBk+g0UP5WUcJgL+FaC0k4Lpk7mvd3ZKuVLObyRyEgNnsTLS
+sgEUNBbaQ+y5XTV7O6VsLWQRNUQiSvy3bOPh+jYellTaJA1XVdM/07dGrVuWmAY+
+wAi8i/NIJurDXhasU8e6Hj/RnQzYVXweYgbuH31c/9RTaU3ndjEBBUE4jqImsYbs
+zG5HxLo154CUzH88YsCf5ZN6Jzl3GLo+213Y0dGT7EHzDpxDtLPzJuo8lnOEqR4/
+5RistNF1fVGl4zma5pUk8ZerRT/mOPm+xAbWXeR6nJEXNI69g3ixPLOdM44jqHbo
+XIpB4FQWJJAtrOLJqM65CPDhWIGs6fvG2U7tv66XVvrUKTBOCfOceGyJEGyKWGX6
+kDECEWFTOdnalsGhX3PUriejhbCNQbkoK6a9PcRIM/VaK++I9eaytz1lShB03Je2
+uXBtzf3uTlc2e/2WZvMmBictoXEZ22WkJh8dDRaKcuxfjkft3DovbcNgP1Yl332b
+oumpQa/FcsdpABEBAAGJAbwEGAEKACYWIQRcJr/sbCRmpSjVVRzQWsdPaJduUgUC
+ZMtrIQIbDAUJA8JnAAAKCRDQWsdPaJduUi+dDAC3mm2EvHXUYxGPvYb6GqfqseDQ
+4rba+jjD6TUaAJVitpTY3CoKKBrdEWRZPR5+WSS/Hx75uAVS84g+3AZza5O4f100
+spZkQtmOxQ4lHVtSZgaj/DR1dgvunHF5bqbiIq1VNOvRGcXafxhKvCksM/aKST6f
+lxYeC8RQAaK8QUJCLqGBR6aiCkFa4/+t95rUEEFrRuukYXm77cfhFlWfiDk5QB5a
+gBjYVCa6INCaEw57dqB3CxBQloIZ4m7PY0ofYiKVD+PWidwztbfO4ee4MeaXVo6z
+NcUVH+fzcCZGgDO1sTuDOzOAdx5uvPmpyzMXzDAe5VYgLoHrsjDhJWsB9YZZjKhx
+/PN+YS3KGsrD8HMgNhAdn3/QkLD2OKl3/sW5kmuFmlywdLkocl8FmZlGyRd/XxVD
+u374UjAHT4NkQzbNV9DPKFeWk8yQq/xVj7wSezndFb/+YuGyz5Skji7mKTuoCjWS
+pawty5DgDQwzHHdhqGlT7Dnb/R9QMEyPvK12HaQ=
+=dt9F
+-----END PGP PUBLIC KEY BLOCK-----
+```
+
+Simply copy that entire text, save it as key.txt, and open a terminal.
+
+In the terminal, use the cd command to navigate to the directory path where you’ve saved key.txt. Then use the `gpg --import` command.
```

Comment:
In the opened terminal, navigate to the directory where you saved 'key.txt' using the 'cd' command, and then execute the 'gpg --import' command.